lib/prmd/commands/init.rb

```diff
@@ -9,9 +9,21 @@ module Generate
     # Creates a default Prmd::Generator using default templates
     #
     # @return [Prmd::Generator]
-    def self.make_generator
+    def self.make_generator(options)
       base = Prmd::Template.load_json('init_default.json')
-      template = Prmd::Template.load_template('init_resource.json.erb', '')
+      template_name = options.fetch(:template) do
+        abort 'render: Template was not provided'
```

Comment:
That appears to not even the case. The doc command does something almost exactly like this for the template option which is also not required there. I tested both with the template option and without and it worked for me. Didn't dig too deep into OptionsParser and instead just followed the precedent that was already established.